.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -245,6 +248,9 @@ jobs:
           - name: Ubuntu 22.04 (gcc) Qt6
             os: ubuntu-22.04
             qt_version: 6
+            aqt_version: '6.7.3'
+            arch: 'gcc_64'
```

Comment:
looks like this is now maybe `linux_gcc_64` for Qt 6 ? Qt 5 is still `gcc_64` according to https://ddalcino.github.io/aqt-list-server/